nav2_controller/src/controller_server.cpp

```diff
@@ -400,9 +403,14 @@ void ControllerServer::computeControl()
       }
 
       if (!loop_rate.sleep()) {
-        RCLCPP_WARN(
-          get_logger(), "Control loop missed its desired rate of %.4fHz",
-          controller_frequency_);
+        real_frequency =
```

Comment:
This does not appear to be following the styling of the project (I see indents with 4 spaces, things being aligned pythonically, etc. 

Please lint this with ament_uncrustify / ament_cpplint for styling